src/app/components/registration-form/registration-form.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, OnInit } from '@angular/core';
+import { NgForm } from '@angular/forms';
+import { DataService } from 'src/app/data/data.service';
+import { User } from 'src/app/data/user';
+
+@Component({
+    selector: 'app-registration-form',
+    templateUrl: './registration-form.component.html',
+    styleUrls: ['./registration-form.component.scss'],
+})
+export class RegistrationFormComponent implements OnInit {
+    user: User = {
+        username: '',
+        email: '',
+        password: '',
+    };
+
+    constructor(private dataService: DataService) {}
+
+    // eslint-disable-next-line @angular-eslint/no-empty-lifecycle-method
+    ngOnInit(): void {}
```

Comment:
You don't really need this here, so there's no reason to disable eslint if you can remove it. You're fixing the issue incorrectly.

Reply:
So if I googled well, there is a constructor or ngOnInit needed. so I deleted ngOnInit.